ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -316,6 +317,51 @@ const GENE_DISEASE_DETAIL_SECTIONS = [
 
 const RNA_SEQ_DETAIL_FIELDS = ['zScore', 'pValue', 'pAdjust']
 
+const INDIVIDUAL_NAME_COLUMN = { name: 'individualName', content: '', format: ({ individualName }) => (<b>{individualName}</b>) }
+
+const RNA_SEQ_COLUMNS = [
+  INDIVIDUAL_NAME_COLUMN,
+  ...RNA_SEQ_DETAIL_FIELDS.map(name => (
+    { name, content: camelcaseToTitlecase(name).replace(' ', '-'), format: row => row[name].toPrecision(3) }
+  )),
+]
+
+const PHENOTYPE_GENE_INFO_COLUMNS = [
+  INDIVIDUAL_NAME_COLUMN,
+  {
+    name: 'diseaseName',
+    content: 'Disease',
+    format: ({ diseaseName, diseaseId }) => (
+      <div>
+        {diseaseName}
+        <br />
+        <i>{diseaseId}</i>
+      </div>
+    ),
+  },
+  { name: 'rank', content: 'Rank' },
+  {
+    name: 'scores',
+    content: 'Scores',
+    format: ({ scores }) => Object.keys(scores).sort().map(scoreName => (
+      <div key={scoreName}>
+        <b>{camelcaseToTitlecase(scoreName).replace(' ', '-')}</b>
+        : &nbsp;
+        { scores[scoreName].toPrecision(3) }
+      </div>
+    )),
+  },
+]
+
+const getDataTable = props => (
```

Comment:
It is really not good practice in react to have functions that return rendered components, its better to have it be a component and render it directly. In this case however, since all you are doing is adding a few props to the component, the cleanest way to do this would be to define a styled component with `attrs`. So something like 
```
const HoverDetailTable = styled(DataTable).attrs({ basic: 'very', compact: 'very', singleLine: true })
```

Reply:
Styling DataTable is not working on my first try. But I found it simpler to define a common prop constant.